docs-devsite/ai.functioncall.md

```diff
@@ -33,6 +34,18 @@ export interface FunctionCall
 args: object;
 ```
 
+## FunctionCall.id
```

Comment:
AFAIK - we don't have this documented in our function calling guide?  Should we?